libsofia-sip-ua/nta/nta.c

```diff
@@ -6466,14 +6480,8 @@ static int nta_incoming_response_headers(nta_incoming_t *irq,
     clone = 1, sip->sip_call_id = sip_call_id_copy(home, irq->irq_call_id);
   if (!sip->sip_cseq)
     clone = 1, sip->sip_cseq = sip_cseq_copy(home, irq->irq_cseq);
-  if (!sip->sip_via) {
```

Comment:
this part of the change seems unrelated and should be in a separate patch with at least details of why we are making this change

Reply:
https://github.com/signalwire/freeswitch/issues/319

Reply:
removed